packages/preview/muw-touying-community/0.1.0/lib.typ

```diff
@@ -0,0 +1,501 @@
+  let subtitle = args.named().at("subtitle", default: auto)
+  
+  if title == auto {
+    title = utils.display-current-heading(level: 2)
+  }
+
+  if title != auto and title != none {
+    self.store.title = title
+  }
+
+  if subtitle == auto {
+    subtitle = utils.display-current-heading(level: 3)
+  }
+
+  if subtitle != auto and subtitle != none {
+    self.store.subtitle = subtitle
+  }
+ 
+  let slide-content = {
+    // set align(left + top)
+    set page(fill: muw_colors.colors.white)
+    set text(fill: muw_colors.colors.black)
+    set par(justify: true, spacing: 0.9em, leading: 0.9em, first-line-indent: 0pt)
+
+    // Hide level 3 headings since they're used as subtitles
+    show heading.where(level: 3): none
+    
+    // Display title
+    if title != auto and title != none {
+      block(
+        inset: (left: 0.5em),
+        text(size: 27pt, weight: "regular", font: "georgia", fill: muw_colors.colors.dunkelblau, title)
+      )
+    }
+    
+    // Display subtitle if available
+    if subtitle != auto and subtitle != none {
+      block(
+        inset: (left: 0.5em),
+        text(size: 20pt, weight: "regular", font: "georgia", fill: muw_colors.colors.hellblau, subtitle)
+      )
+    }
+    
+    set par(justify: true, 
+            leading: 1.3em, 
+            first-line-indent: 0pt)
+
+    // Display slide content
+    if body != none {
+      set text(size: 17pt, weight: "regular", font: "lucida sans", fill: muw_colors.colors.black)
+      block(
+        inset: (top: 0.5em, left: 0.5em, right: 0.5em),
+        body
+      )
+    }
+  }
+
+  touying-slide(self: self, slide-content)
+})
+////////////////////////////////////////////////////////////////
+
+ 
+// Black slide for medical imaging
+// Imaging slide with black background for medical images
+#let imaging-slide(
+  title: none,
+  subtitle: none,
+  image-path: none,
+  image-width: 100pt,  // Changed to absolute value
+  ..args
+) = touying-slide-wrapper(self => {
+  let body = args.pos().at(0, default: none)
+  
+  let slide-content = {
+    set align(left + top)
+    set page(fill: muw_colors.colors.black)
+    set text(fill: muw_colors.colors.white)
+    set par(justify: true, leading:0.9em, first-line-indent: 0pt)
+    
+    // Title
+    if title != none {
+      block(
+        inset: (left: 0.5em),
+        text(size: 27pt, weight: "regular", fill: muw_colors.colors.white, title)
+      )
+    }
+    
+    // Subtitle
+    if subtitle != none {
+      block(
+        inset: (left: 0.5em),
+        text(size: 20pt, weight: "regular", fill: muw_colors.colors.hellblau, subtitle)
+      )
+    }
+    
+    // Main content area with image and text side by side
+    if image-path != none or body != none {
+      grid(
+        columns: (image-width, 1fr),  
+        column-gutter: 1em,
+        
+        // Left column: Image
+        if image-path != none {
+          align(left + top)[
+            #image(image-path, width: 100%)
```

Comment:
This path will be resolved relatively to the package root, not to the user's project root. You should take an `image` as a parameter directly, instead of just a file path. You can still configure the default width using a `set` rule. [More details here](https://github.com/typst/packages/blob/main/docs/resources.md#paths-are-package-scoped).